src/frontend/js/widgets/Dashboard/utils/learnerRouteMessages.tsx

```diff
@@ -5,71 +5,17 @@ import { defineMessages } from 'react-intl';
-  PREFERENCES_ADDRESS_CREATION = '/preferences/addresses/create',
-  PREFERENCES_CREDIT_CARD_EDITION = '/preferences/credit-cards/:creditCardId',
+  PREFERENCES_ADDRESS_EDITION = `${PREFERENCES}/addresses/:addressId`,
+  PREFERENCES_ADDRESS_CREATION = `${PREFERENCES}/addresses/create`,
+  PREFERENCES_CREDIT_CARD_EDITION = `${PREFERENCES}/credit-cards/:creditCardId`,
 }
 
-// Translations of dashboard route paths
-export const LEARNER_DASHBOARD_ROUTE_PATHS = defineMessages<LearnerDashboardPaths>({
-  [LearnerDashboardPaths.COURSES]: {
-    id: 'components.Dashboard.DashboardRoutes.courses.path',
-    description: 'The path to display the courses view.',
-    defaultMessage: '/courses',
-  },
-  [LearnerDashboardPaths.ORDER]: {
-    id: 'components.Dashboard.DashboardRoutes.order.path',
-    description: 'The path to display an order detail view.',
-    defaultMessage: '/courses/orders/{orderId}',
-  },
-  [LearnerDashboardPaths.ORDER_RUNS]: {
-    id: 'components.Dashboard.DashboardRoutes.order.runs.path',
-    description: 'The path to display an order runs view.',
-    defaultMessage: '/courses/orders/{orderId}/runs',
-  },
-  [LearnerDashboardPaths.COURSE]: {
-    id: 'components.Dashboard.DashboardRoutes.course.path',
-    description: 'The path to display a course detail view.',
-    defaultMessage: '/courses/{code}',
-  },
-  [LearnerDashboardPaths.CERTIFICATES]: {
-    id: 'components.Dashboard.DashboardRoutes.certificates.path',
-    description: 'The path to display the certificates view.',
-    defaultMessage: '/certificates',
-  },
-  [LearnerDashboardPaths.CONTRACTS]: {
-    id: 'components.Dashboard.DashboardRoutes.contracts.path',
-    description: 'The path to display the training contracts view.',
-    defaultMessage: '/training-contracts',
-  },
-  [LearnerDashboardPaths.PREFERENCES]: {
-    id: 'components.Dashboard.DashboardRoutes.preferences.path',
-    description: 'The path to display the preferences view.',
-    defaultMessage: '/preferences',
-  },
-  [LearnerDashboardPaths.PREFERENCES_ADDRESS_EDITION]: {
-    id: 'components.Dashboard.DashboardRoutes.preferences.addresses.edition.path',
-    description: 'The path to display the addresses edition view.',
-    defaultMessage: '/preferences/addresses/{addressId}',
-  },
-  [LearnerDashboardPaths.PREFERENCES_ADDRESS_CREATION]: {
-    id: 'components.Dashboard.DashboardRoutes.preferences.addresses.creation.path',
-    description: 'The path to display the addresses creation view.',
-    defaultMessage: '/preferences/addresses/create',
-  },
-  [LearnerDashboardPaths.PREFERENCES_CREDIT_CARD_EDITION]: {
-    id: 'components.Dashboard.DashboardRoutes.preferences.creditCards.edition.path',
-    description: 'The path to display the credit cards edition view.',
-    defaultMessage: '/preferences/credit-cards/{creditCardId}',
-  },
-});
-
 // Translations of dashboard route labels
 export const LEARNER_DASHBOARD_ROUTE_LABELS = defineMessages<LearnerDashboardPaths>({
```

Comment:
Proposal:  
Move `LearnerDashboardPaths` into `utils.learnerRoutes.tsx`
This way `learnerRoutes` will include routing logic and `learnerRouteMessages` will includes translations logic.

Reply:
Personaly I prefer having both data ( path + labels ) within the same file it avoid having to go back and forth between two files, democratos @jbpenrath?

I think it is good to have the opinions of everyone on these choices as they are related to architecture

Reply:
Personally I do not like to have to navigate between too many modules so IMO here this single module is fine, it gathers all learners routing logic.